flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -36,13 +36,32 @@ jobmanager.rpc.address: localhost
 
 jobmanager.rpc.port: 6123
 
+# The host interface the JobManager will bind to. My default, this is localhost, and will prevent
+# the JobManager from communicating outside the machine/container it is running on.
+#
+# To enable this, set the bind-host address to one that has access to an outside facing network
+# interface, such as 0.0.0.0.
+
+jobmanager.bind-host: localhost
+
 
 # The total process memory size for the JobManager.
 #
 # Note this accounts for all memory usage within the JobManager process, including JVM metaspace and other overhead.
 
 jobmanager.memory.process.size: 1600m
 
+# The external address of the network interface where the TaskManager is exposed.
+
+taskmanager.host: localhost
```

Comment:
This shouldn't be necessary; this is just under which address other processes try to talk to it. If it isn't set then we'll figure that out on our own.

Reply:
But it shouldn't have any effect on _whether_ the TM is reachable from the outside or not.